cmd/main.go

```diff
@@ -0,0 +1,49 @@
+// Package cmd contains all command-line interface commands for Trellis CLI.
+package cmd
+
+import (
+	"fmt"
+	"runtime"
+
+	"github.com/mitchellh/cli"
+	"github.com/roots/trellis-cli/pkg/lima"
+	"github.com/roots/trellis-cli/pkg/vm"
+	"github.com/roots/trellis-cli/trellis"
+)
+
+// This file provides centralized declarations of functions as variables
+// to enable mocking in tests and prevent duplicate implementations.
+// Previously, these functions were defined in multiple places, causing
+// "redeclared in this block" errors during compilation.
```

Comment:
Can we avoid all these changes for now? This is a whole new pattern that I'm not sure about yet

I'm okay if this limits the testability temporarily